src/components/entity-panel/sections/tactic/TheTacticListContent.vue

```diff
@@ -2,55 +2,85 @@
   <span v-if="!items.length" class="caption px-2" v-text="$t('tactic.noTacticsFound')"/>
   <v-treeview
     v-else
-    v-model="activeElements"
+    :active="[currentTacticId]"
     :items="items"
-    :open="open"
     class="pa-0"
     dense
     open-on-click
     activatable
+    @update:active="switchTactic($event)"
```

Comment:
im pretty sure you can just do `@update:active="switchTactic"` and the event will be passed automatically

Reply:
Yeah of course you can, I was doing something else at the time and thought I needed another property passing in, but obviously I don't.

I have removed.